usaspending_api/local_settings.example.py

```diff
@@ -0,0 +1,9 @@
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.postgresql',
+        'NAME': 'usaspending_api',
+        'USER': 'usaspending_api',
+        'PASSWORD': '',
+    }
+}
```

Comment:
how do people generally manage this locally? autoenv or virtualenvwrapper post activate hooks?